server.js

```diff
@@ -115,10 +126,11 @@ utils.addNunjucksFilters(nunjucksAppEnv)
 app.set('view engine', 'html')
 
 // Middleware to serve static assets
-app.use('/public', express.static(path.join(projectDir, '/public')))
+app.use('/public', express.static(path.join(projectDir, 'public')))
+app.use('/assets', express.static(path.join(projectDir, 'app', 'assets')))
```

Comment:
We'll need to update the documentation in https://github.com/alphagov/govuk-prototype-kit-docs/blob/main/docs/documentation/adding-css-javascript-and-images.md to reflect this change.

Reply:
Ah cool, https://github.com/alphagov/govuk-prototype-kit/pull/1533/commits/e3bac4f7b418b10f68df01733d3aa3f8e977cf80#diff-a4c65ede64197e1a112899a68bf994485b889c4b143198bac4af53425b38406f makes it so that assets are still served from `/public`, so documentation stays valid.